packetbeat/etc/beat.yml

```diff
@@ -7,171 +7,172 @@
-    # json output.
-    # possible values:
-    #    maxvalue: -1  # store all values (text based protocol multi-get)
-    #    maxvalue: 0   # store no values at all
-    #    maxvalue: N   # store up to N values
-    # Default: 0
-    # maxvalues: -1
-
-    # Use maxbytespervalue to limit the number of bytes to be copied per value element.
-    # Note: Values will be base64 encoded, so actual size in json document
-    #       will be 4 times maxbytespervalue.
-    # Default: unlimited
-    # maxbytespervalue: 100
-
-    # UDP transaction timeout in milliseconds.
-    # Note: Quiet messages in UDP binary protocol will get response only in error case.
-    #       The memcached analyzer will wait for udptransactiontimeout milliseconds
-    #       before publishing quiet messages. Non quiet messages or quiet requests with
-    #       error response will not have to wait for the timeout.
-    # Default: 200
-    # udptransactiontimeout: 1000
-
-  mysql:
-    # Configure the ports where to listen for MySQL traffic. You can disable
-    # the MySQL protocol by commenting out the list of ports.
-    ports: [3306]
-
-  pgsql:
-    # Configure the ports where to listen for Pgsql traffic. You can disable
-    # the Pgsql protocol by commenting out the list of ports.
-    ports: [5432]
-
-  redis:
-    # Configure the ports where to listen for Redis traffic. You can disable
-    # the Redis protocol by commenting out the list of ports.
-    ports: [6379]
-
-  thrift:
-    # Configure the ports where to listen for Thrift-RPC traffic. You can disable
-    # the Thrift-RPC protocol by commenting out the list of ports.
-    ports: [9090]
-
-  mongodb:
-    # Configure the ports where to listen for MongoDB traffic. You can disable
-    # the MongoDB protocol by commenting out the list of ports.
-    ports: [27017]
-
-  nfs:
-    # Configure the ports where to listen for NFS traffic. You can disable
-    # the NFS protocol by commenting out the list of ports.
-    ports: [2049]
-
-############################# Processes #######################################
-
-# Configure the processes to be monitored and how to find them. If a process is
-# monitored then Packetbeat attempts to use it's name to fill in the `proc` and
-# `client_proc` fields.
-# The processes can be found by searching their command line by a given string.
-#
-# Process matching is optional and can be enabled by uncommenting the following
-# lines.
-#
-#procs:
-#  enabled: false
-#  monitored:
-#    - process: mysqld
-#      cmdline_grep: mysqld
-#
-#    - process: pgsql
-#      cmdline_grep: postgres
-#
-#    - process: nginx
-#      cmdline_grep: nginx
-#
-#    - process: app
-#      cmdline_grep: gunicorn
+packetbeat:
+
+  ############################# Sniffer #########################################
+
+  # Select the network interfaces to sniff the data. You can use the "any"
+  # keyword to sniff on all connected interfaces.
+  interfaces:
```

Comment:
Was thinking about the same, see: https://github.com/elastic/beats/issues/1488 (Option 2)

Going with the above would mean doing the same for flows, protocols and and procs.